src/meltano/cli/schedule.py

```diff
@@ -281,6 +287,13 @@ def remove(ctx, name):
     ctx.obj["schedule_service"].remove(name)
 
 
+def validate_interval(interval: str):
+    """Validate the interval for a schedule."""
+    if interval not in CRON_INTERVALS and not croniter.is_valid(interval):
+        raise BadCronError(interval)
+    return interval
```

Comment:
You can probably catch an invalid interval earlier by using a [`click.ParamType`](https://click.palletsprojects.com/en/8.1.x/api/#click.ParamType) for the `--interval` option. Wdyt?

Reply:
It works @edgarrmondragon , nice suggestion @ @